.github/CODEOWNERS

```diff
@@ -81,6 +77,16 @@
 /packages/fx-core/tests/plugins/resource/function @eriolchan @hund030 @IvanJobs
 /packages/fx-core/src/plugins/resource/function/utils/depsChecker @a1exwang @kimizhu @meifans @qinezh
 
+/templates/tab @hund030 @eriolchan @IvanJobs
+/templates/function-base @hund030 @eriolchan @IvanJobs
+/templates/function-triggers @hund030 @eriolchan @IvanJobs
+/templates/blazor-base @JerryYangKai @eriolchan @IvanJobs
+/templates/bot @IvanJobs @eriolchan @Siglud @Yukun-dong
```

Comment:
do we still have templates/msgext and templates/bot-msgext?

Reply:
Is that covered in @yukun-dong 's PR?

Reply:
Let's remove them after yukun's PR merged.